src/arch/xtensa/Kconfig

```diff
@@ -17,6 +17,16 @@ config SMP
 	help
 	  Indicates that architecture uses multiple cores
 
+config HAVE_MASTER_CORE_ROM
+	bool "Have master core rom"
+	default y
+
+config HAVE_SLAVE_CORE_ROM
+	bool "Have slave core rom"
+	default y
+	help
+	  Indicates that slave core does not need alternate reset vector
```

Comment:
The help does not align with the option naming, one is for ROM and the other describes a reset vector.

Reply:
Yeap but "Have slave core rom" is self explanatory so I didnt like to repeat this in help, I referenced to alternate reset vector because it is the most important reason (and atm the only one) to turn it on/off.
Changed this help description a little, please check